src/Mod/Path/PathScripts/PostUtilsArguments.py

```diff
@@ -34,6 +34,14 @@
 import os
 import shlex
 
+import FreeCAD
+
```

Comment:
Unused taking line 39 into consideration
```suggestion
```

Reply:
Greed and thank you for the imput!